addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/handler/BridgeV6Handler.java

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.milight.internal.handler;
+
+import java.io.IOException;
+import java.net.InetAddress;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.config.core.Configuration;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.openhab.binding.milight.internal.MilightBindingConstants;
+import org.openhab.binding.milight.internal.protocol.MilightV6SessionManager;
+import org.openhab.binding.milight.internal.protocol.MilightV6SessionManager.ISessionState;
+import org.openhab.binding.milight.internal.protocol.MilightV6SessionManager.SessionState;
+
+/**
+ * The {@link BridgeV6Handler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author David Graeff - Initial contribution
+ */
+@NonNullByDefault
+public class BridgeV6Handler extends AbstractBridgeHandler implements ISessionState {
+    @NonNullByDefault({})
+    private MilightV6SessionManager session;
+    final SimpleDateFormat sdf = new SimpleDateFormat("HH:mm");
```

Comment:
Please note that SimpleDateFormat is not thread-safe, in java 8 [DateTimeFormatter](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html) which is both thread-safe and immutable.